nx/include/switch/kernel/svc.h

```diff
@@ -597,27 +633,65 @@ Result svcGetInfo(u64* out, u32 id0, Handle handle, u64 id1);
 
 ///@}
 
+///@name Cache Management
+///@{
+
+/**
+ * @brief Flushes the entire data cache (by set/way).
+ * @note Syscall number 0x2A.
+ * @warning This is a privileged syscall. Use \ref envIsSyscallHinted to check if it is available.
```

Comment:
@TuxSH Should a note be added saying that this syscall is dangerous and shouldn't be used?